US COVID-19.Rmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "US COVID-19"
+author: "Xing Yanfeng"
+date: "2020/7/20"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = F)
+```
+
+This is graphs of **US COVID-19 for 2020** data to show tendency of case and death increase. Data were downloaded from [*The New York Times*](https://www.nytimes.com/interactive/2020/us/coronavirus-us-cases.html). 
+
+```{r load library}
+library(ggplot2)
+library(readr)
+```
+
+### US COVID-19 data for 2020
+
+
+```{r plotting}
+#importing data
+us_data <- read_csv("us.csv", col_types = "Dii") 
+
+#plotting data
+ggplot(us_data, aes(x = date)) +
+  geom_col(aes(y = cases), color = "#0080FF", fill = "#0080FF", alpha = 0.5) +
+  geom_col(aes(y = deaths), color = "red", fill = "red", alpha = 0.5) +
+#adding title and caption
+  labs(title = "US COVID-19 data for 2020",
+       x = "Date", 
+       y = "Total Cases/Deaths", 
+       caption = "Source: The New York Times") +
+#setting theme
+  theme_classic() +
+  theme(plot.title = element_text(hjust = 0.5)) +
+#setting scales
+  scale_y_continuous(limits = c(0, 3800000), breaks =  seq(0, 3800000, by = 500000)) +
```

Comment:
This is *definitely* a matter of personal preference, but (at times) the really large numbers can be cumbersome to read, especially without the comma separations. It can help a reader a lot when the axis is scaled down.